justfile

```diff
@@ -24,7 +24,7 @@ check *args:
 
 # run pytest for ci; additional arguments are forwarded to pytest
 ci-check *args:
-    poetry run pytest --durations=25 -ra --junitxml=junit.xml --cov=ibis --cov-report=xml:coverage.xml {{ args }}
```

Comment:
What is the default output behaviour without the `-ra` flag? Do we still get stacktraces of fails?

Reply:
Yep. `-ra` shows all the xfails and skips, which IMO aren't that useful in CI

Reply:
Ah! I agree with this.